src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -77,19 +82,24 @@ class PropertyAccessor implements PropertyAccessorInterface
      *                               or self::DISALLOW_MAGIC_METHODS for none
      * @param int $throw             A bitwise combination of the THROW_* constants
      *                               to specify when exceptions should be thrown
+     * @param int $bypassPropertyHooks A bitwise combination of the BYPASS_PROPERTY_HOOK_* constants
```

Comment:
`@param int-mask-of<self::BYPASS_PROPERTY_HOOK_*> $bypassPropertyHooks` so that static analysis tool know about the precise type and can report invalid usages.

Reply:
thx, didn't know that syntax